stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -530,6 +530,23 @@ spec:
 {{ toYaml .Values.filebeat.resources | indent 10 }}
         terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
       {{- end }}
+      {{- if .Values.fluentd.enabled }}
+      - name: {{ .Values.fluentd.name }}
+        image: "{{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.version }}"
+        imagePullPolicy: {{ .Values.fluentd.image.pullPolicy }}
+        volumeMounts:
+        - name: fluent-conf
+          mountPath: /fluent-bit/etc/
+        - name: artifactory-volume
+          mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+        livenessProbe:
+{{ toYaml .Values.fluentd.livenessProbe | indent 10 }}
+        readinessProbe:
+{{ toYaml .Values.fluentd.readinessProbe | indent 10 }}
+        resources:
+{{ toYaml .Values.fluentd.resources | indent 10 }}
+        terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
```

Comment:
Since you put this option, add it also to the template in `values.yaml`. Commented out if not needed.

Reply:
done